pkg/metrics/metrics.go

```diff
@@ -71,71 +68,46 @@ func NewPrometheusNodeConfig(clientset *kubernetes.Clientset) (*PrometheusNodeCo
 	}, err
 }
 
-// Add will create the Services and Service Monitors that allows the operator to export the metrics by using
-// the Prometheus operator
-func Add(ctx context.Context, cfg *rest.Config, namespace string) error {
-	// Add to the below struct any other metrics ports you want to expose.
-	servicePorts := []v1.ServicePort{
-		{Port: Port, Name: PortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: Port}},
-	}
-
-	// Create Service object to expose the metrics port(s).
-	service, err := metrics.CreateMetricsService(ctx, cfg, servicePorts)
+// Validate will verify if the Services and Service Monitors are created in the operator namespace
+func Validate(ctx context.Context, cfg *rest.Config, namespace string) error {
```

Comment:
Why are you validating this here? Shouldn't this be an e2e test?

Reply:
we already have an e2e test for this, it was just to make sure the resources exist before doing anything but it is not required to do all the validations here hence I removed it.